packages/react/src/toast/useToastManager.ts

```diff
@@ -0,0 +1,117 @@
+'use client';
+import * as React from 'react';
+import { ToastContext } from './provider/ToastProviderContext';
+
+/**
+ * Returns the array of toasts and methods to create toasts.
```

Comment:
It's not just creating

```suggestion
 * Returns the array of toasts and methods to manage them.
```